pkg/config/issues.go

```diff
@@ -79,13 +79,25 @@ var DefaultExcludePatterns = []ExcludePattern{
 	},
 	{
 		ID:      "EXC0012",
-		Pattern: "exported (method|function|type|const) (.+) should have comment or be unexported",
+		Pattern: `exported (.+) should have comment or be unexported`,
```

Comment:
Unfortunately, this pattern does not match the actual message from `revive`.
```
❯ golangci-lint --version
golangci-lint has version 1.41.0 built from eed344f6 on 2021-06-16T19:58:50Z

❯ golangci-lint run
main.go:16:2: exported: exported const XXX should have comment (or a comment on this block) or be unexported (revive)
	XXX = 0.1
```

@ldez Could you check that again?

Reply:
It's a specific case for `const` in a group, I will fix that.

Reply:
fixed in #2064